setup.py

```diff
@@ -4,7 +4,7 @@
 
 setuptools.setup(
     name='jsonpath-ng',
-    version='1.5.3',
+    version='1.5.3.1',
```

Comment:
See my previous comment about the version change.

Reply:
Done & understood, just needed this on my branch to easily differentiate between versions.